modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-request-info-contact/index.js

```diff
@@ -29,19 +29,32 @@ const getContactInfo = async () => {
 				},
 			}
 		);
+
 		if (response.ok) {
 			const data = await response.json();
 			const firstName = data?.r_usrToMDFReqs_user?.givenName;
+			const lastName = data?.r_usrToMDFReqs_user?.alternateName;
 			const infoEmail = data?.r_usrToMDFReqs_user?.emailAddress;
+			const telephone =
+				data?.r_usrToMDFReqs_user?.userAccountContactInformation
+					.telephones[0].phoneNumber;
```

Comment:
Hey @caiosouza15 , @jairmedeiros 

In case of the user having multiple phones, we are getting the first one, it makes sense to display all the numbers?

```suggestion
			const telephones =
				data?.r_usrToMDFReqs_user?.userAccountContactInformation
					.telephones.map(({phoneNumber}) => phoneNumber).join(", ");
```